static/src/javascripts/projects/common/views/acquisitions-epic-testimonials-prominent.html

```diff
@@ -0,0 +1,41 @@
+<div class="contributions__epic" data-component="<%=componentName%>">
+    <div>
+        <h2 class="contributions__title contributions__title--epic">
+            Why our readers are supporting us:
+        </h2>
+        <div class="epic__testimonial__container">
+            <p class="epic__testimonial__text">
```

Comment:
Could you convert all of these into `<blockquote />` elements, because they provide proper semantics :)